ts/features/itwallet/presentation/remote/machine/machine.ts

```diff
@@ -27,16 +37,51 @@ export const itwRemoteMachine = setup({
           actions: assign(({ event }) => ({
             payload: event.payload
           })),
-          target: "RemoteRequestValidation"
+          target: "PayloadValidation"
         }
       }
     },
-    RemoteRequestValidation: {
+    PayloadValidation: {
       description: "Validating the remote request payload before proceeding",
-      tags: [ItwTags.Loading]
+      tags: [ItwTags.Loading],
+      always: [
+        {
+          guard: not("isWalletActive"),
+          actions: assign({
+            failure: {
+              type: RemoteFailureType.WALLET_INACTIVE,
+              reason: "IT Wallet is inactive"
+            }
+          }),
+          target: "Failure"
+        },
+        {
+          target: "PayloadValidated"
+        }
+      ]
+    },
+    PayloadValidated: {
+      description: "The remote request payload has been validated",
+      on: {
+        close: {
+          actions: "closeIssuance"
+        }
+      }
     },
     Failure: {
-      description: "This state is reached when an error occurs"
+      entry: "navigateToFailureScreen",
+      description: "This state is reached when an error occurs",
+      on: {
+        "accept-tos": {
+          actions: "navigateToTosScreen"
+        },
```

Comment:
User is not accepting any ToS here.

```suggestion
        "go-to-wallet-activation": {
          actions: "navigateToDiscoveryScreen"
        },
```